pkg/kubectl/scale.go

```diff
@@ -48,7 +48,7 @@ func ScalerFor(kind unversioned.GroupKind, c client.Interface) (Scaler, error) {
 	case extensions.Kind("ReplicaSet"):
 		return &ReplicaSetScaler{c.Extensions()}, nil
 	case extensions.Kind("Job"), batch.Kind("Job"):
```

Comment:
Otherwise LGTM, thanks. Just want to make sure I understand it correctly before merging.